tests/e2e/e2e_routesv2_permissive_test.go

```diff
@@ -0,0 +1,115 @@
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	. "github.com/openservicemesh/osm/tests/framework"
+)
+
+// TODO : remove as a part of routes refactor (#2397). Routes v2 will become the default and this test will no longer be needed
+var _ = OSMDescribe("Routes V2 Permissive Traffic Policy Mode",
+	OSMDescribeInfo{
+		Tier:   2,
+		Bucket: 2,
```

Comment:
Bucket 3 to distribute load?
```suggestion
		Bucket: 3,
```